apps/src/applab/applab.js

```diff
@@ -512,7 +513,14 @@ Applab.getHtml = function () {
 };
 
 Applab.setLevelHtml = function (html) {
-  Applab.levelHtml = designMode.addScreenIfNecessary(html);
+  if (html === '') {
+    if (window.dashboard) {
```

Comment:
I'm not a big fan of this pattern of reaching out to the dashboard, but I think this is temporary, right?